velero-csi-plugin/csi_snapshotter.go

```diff
@@ -55,42 +62,58 @@ func (p *CSISnapshotter) AppliesTo() (velero.ResourceSelector, error) {
 func (p *CSISnapshotter) Execute(item runtime.Unstructured, backup *v1.Backup) (runtime.Unstructured, []velero.ResourceIdentifier, error) {
 	p.log.Info("CSISnapshotter Execute")
 
+	// Do nothing if volume snapshots have not been requested in this backup
+	if !*backup.Spec.SnapshotVolumes {
```

Comment:
need to nil-check this. we have a `pkg/util/boolptr` in velero, if you want to use it